### DISCUSSION TITLE
Renovate aws AMI

### DISCUSSION THREAD

**Discussion by Mehmoodhassan** [renovatebot/renovate#17059] [Request Help]
Hi, I'm building a custom ami using packer and right now I have Ubuntu 20.04 version of ami. I want to use renovate to upgrade my ami would you please help me in this.

**Comment by PhilipAbed** [comment]
i'm not familiar with packer or ami but here's how renovate works in general: 
1) renovate scans dependencies + versions in files 
so you need a packages file to scan
if we dont support it, you can use Regex Managers and add it to the file matches
2) renovate will look in the datasource for new versions, what kind of dependencies does your file have?  lets say we find the words inside a file with python dependencies like this this:
django 3.2.1
then your datasource  is python 

renovate will magically do the rest for you 
here's some pointers to start with:
https://github.com/renovatebot/tutorial
will let u understand how renovate works

and see examples in github for regex managers
https://github.com/search?q=regexmanagers+in%3Afile+filename%3Arenovate.json
